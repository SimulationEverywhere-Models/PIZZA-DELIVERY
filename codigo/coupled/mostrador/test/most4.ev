00:00:01:000 abrir_m 1
00:00:02:000 atiende_m 2
00:00:15:000 atiende_m 3
00:00:32:000 recibe_ent_m 2
00:00:36:000 moto_ocupada_m 1
00:00:38:000 atiende_m 4
00:00:43:000 recibe_ent_m 4
00:00:52:000 moto_libre_m 1
00:00:55:000 recibe_ent_m 3
00:00:60:000 cerrar_m 1
