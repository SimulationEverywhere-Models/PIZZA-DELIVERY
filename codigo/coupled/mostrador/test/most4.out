00:00:01:000 lista_ll_m 1
00:00:02:000 ocupado_m 1
00:00:04:000 libre_m 1
00:00:04:030 manda_pedido 2
00:00:15:000 ocupado_m 1
00:00:17:000 libre_m 1
00:00:17:030 manda_pedido 3
00:00:32:000 ocupada_ent_m 1
00:00:32:030 dist_out_m 2
00:00:32:030 libre_m_ent 1
00:00:38:000 ocupado_m 1
00:00:40:000 libre_m 1
00:00:40:030 manda_pedido 4
00:00:43:000 ocupada_ent_m 1
00:00:52:000 dist_out_m 4
00:00:52:000 libre_m_ent 1
00:00:55:000 ocupada_ent_m 1
00:00:55:030 dist_out_m 3
00:00:55:030 libre_m_ent 1
00:01:00:000 cerrado_m 1
